hack/label_sync/labels.yaml

```diff
@@ -0,0 +1,108 @@
+      description: Categorizes issue or PR as related to a bug.
+      color: ee0701
+    - name: kind/cleanup
+      description: Categorizes issue or PR as related to cleaning up code, process, or technical debt.
+      color: c7def8
+    - name: kind/documentation
+      description: Categorizes issue or PR as related to documentation.
+      color: c7def8
+    - name: kind/feature
+      description: Categorizes issue or PR as related to a new feature.
+      color: c7def8
+    # Priority
+    - name: priority/awaiting-more-evidence
+      description: Lowest priority. Possibly useful, but not yet enough support to actually get it done.
+      color: fef2c0
+    - name: priority/backlog
+      description: Higher priority than priority/awaiting-more-evidence.
+      color: ffe699
+    - name: priority/blocked
+      description: Currently blocked by another task, issue or PR.
+      color: d93f0b
+    - name: priority/critical-urgent
+      description: Highest priority. Must be actively worked on as someone's top priority right now.
+      color: e60000
+    - name: priority/important-longterm
+      description: Important over the long term, but may not be staffed and/or may need multiple sprints to complete.
+      color: ffb3b3
+    - name: priority/important-soon
+      description: Must be staffed and worked on either currently, or very soon, ideally in time for the next sprint.
+      color: ff6666
+    # Size
+    - name: size/L
+      description: Requires few days to complete the PR or the issue.
+      color: 1a5587
+      addedBy: humans
+    - name: size/M
+      description: Requires about a day to complete the PR or the issue.
+      color: 216fb3
+      addedBy: humans
+    - name: size/S
+      description: Requires less than a day to complete the PR or the issue.
+      color: 93c2ea
+      addedBy: humans
+    - name: size/XL
+      description: Requires about a week to complete the PR or the issue.
+      color: 5a55a1
+      addedBy: humans
+    - name: size/XS
+      description: Requires less than an hour to complete the PR or the issue.
+      color: cceef5
+      addedBy: humans
+    - name: size/XXL
+      description: Requires more than a week to complete the PR or the issue.
+      color: 6f048d
+      addedBy: humans
+    # Needs
+    - name: needs-kind
+      description: Indicates a PR or issue lacks a `kind/foo` label and requires one.
+      color: ededed
+      target: prs
+      prowPlugin: require-matching-label
+      addedBy: prow
+    - name: needs-priority
+      description: Indicates a PR or issue lacks a `priority/foo` label and requires one.
+      color: ededed
+      target: prs
+      prowPlugin: require-matching-label
+      addedBy: prow
+    - name: needs-size
+      description: Indicates a PR or issue lacks a `size/foo` label and requires one.
+      color: ededed
+      target: prs
+      prowPlugin: require-matching-label
+      addedBy: prow
+repos:
+  3scale-ops/marin3r:
```

Comment:
Is that OK to have an explicit repo from `3scale-ops` org? I thought it was done by org (`3scale-ops`) or explicit repos (`3scale/platform`, `3scale/3scale-saas`).

Reply:
That's because marin3r requires the `ok-to-test` label to trigger the test workflow, to avoid external PRs triggering the test pipeline without the explicit ok from a reviewer.

Reply:
Ah ok, that's make sense!

Reply:
That's to apply some labels that only applies to `marin3r`. The `org`/`repo` scope limitation is only in the labeler app:

https://github.com/3scale-ops/test-infra/blob/main/.github/workflows/label-sync.yaml#L25
```
❯ docker run gcr.io/k8s-prow/label_sync:latest --help 2>&1 | egrep -A1 '(-only|-org)'
  -only string
    	Only look at the following comma separated org/repos
--
  -orgs string
    	Comma separated list of orgs to sync
```

````
❯ make run-both
docker run \
                --rm \
                --volume "/Users/rael/Code/gh/3scale-ops/test-infra/hack/label_sync/secrets/oauth":/etc/github/oauth \
                --volume "/Users/rael/Code/gh/3scale-ops/test-infra/hack/label_sync/labels.yaml":/etc/config/labels.yaml \
                gcr.io/k8s-prow/label_sync:latest \
                        --config=/etc/config/labels.yaml \
                        --confirm=true \
                        --orgs=3scale-ops \
                        --only=3scale/platform,3scale/3scale-saas \
                        --token=/etc/github/oauth
{"component":"label_sync","file":"label_sync/main.go:735","func":"main.main","level":"fatal","msg":"--only and --orgs cannot both be set","severity":"fatal","time":"2021-04-14T09:50:47Z"}
make: *** [run-both] Error 1
```